test/metadata/BuildDropdown.test.tsx

```diff
@@ -3,52 +3,49 @@ import { Provider } from "react-redux";
 import { render, RenderResult, screen } from "@testing-library/react";
 import userEvent from "@testing-library/user-event";
 import { store } from "../../src/store";
-import { Build } from "../../src/features/metadata/components/BuildDropdown";
+import { BuildDropdown } from "../../src/features/metadata/components/BuildDropdown";
+import { mockBuilds } from "../../src/features/metadata/mocks/mockBuilds";
 import { mockTheme } from "../testutils";
-
-const builds = [
-  {
-    id: 0,
-    name: "August 5th, 2022 - 4:04 - Available",
-    status: "Building"
-  },
-  {
-    id: 1,
-    name: "August 5th, 2022 - 3:57 - Available",
-    status: "Building"
-  }
-];
+import { buildDatetimeStatus } from "../../src/utils/helpers/buildMapper";
 
 describe("<BuildDropdown />", () => {
   let component: RenderResult;
+  const currentBuildId = 1;
 
   beforeEach(() => {
     component = render(
       mockTheme(
         <Provider store={store}>
-          <Build
-            builds={builds}
-            selectedBuildId={0}
-            currentBuildStatus="Building"
+          <BuildDropdown
+            builds={mockBuilds}
+            currentBuildId={1}
+            selectedBuildId={1}
           />
         </Provider>
       )
     );
   });
 
   it("should render component", () => {
-    expect(component.container).toHaveTextContent(builds[0].name);
+    expect(component.container).toHaveTextContent(
+      buildDatetimeStatus(mockBuilds[0], currentBuildId)
+    );
   });
 
   it("should display builds in the dropdown", async () => {
+    // This is what the dropdown should display for the build
+    const dropdownOptionName = buildDatetimeStatus(
```

Comment:
Feels a little funky to me to call a function here rather than to just hard-code the string, perhaps I should go back and update this

Reply:
Well, if your hesitation is that `BuildDropdown` uses `buildDatetimeStatus` to create the string, remember that you're already testing `buildDatetimeStatus` elsewhere. So I think this is fine, this test is really checking that the correct text appears in the element, not that `buildDatetimeStatus` makes the right string. Anyway, if you'd prefer this test looks different I'll defer to you, but IMO this is fine.